README.md

```diff
@@ -54,7 +54,7 @@ You may also find help on these community resources:
 
 ## Roadmap
 
-The [SBT for AWS Roadmap project board](PLACEHOLDER) lets developers know about our upcoming features and priorities to help them plan how to best leverage the CDK and identify opportunities to contribute to the project. 
+The [SBT for AWS Roadmap project board](https://github.com/orgs/awslabs/projects/167) lets developers know about our upcoming features and priorities to help them plan how to best leverage the CDK and identify opportunities to contribute to the project. 
```

Comment:
we recently discovered that this is not public.

Reply:
This change is only for internal for now. Update it and wait for being public, if a new URL will be needed, we'll update it.
And Korean ReadMe applied the latest version.